### ISSUE TITLE
Websocket Stop

### ISSUE THREAD

**Issue by emin-karadag** [ridicoulous/Bitmex.Net#4]
Hello, I want to stop listening to websocket, but I was not successful. Can you help me please?

I tried as follows but it didn't work!

```
await _socketClient.UnsubscribeAll().ConfigureAwait(false);
_socketClient.Dispose();
```

**Comment by ridicoulous** [comment]
Unfortunately this functionality was not well tested yet. PR are welcome, or you can wait when I’ll have free time to fix it.

**Comment by emin-karadag** [comment]
Hello, First of all, thank you for your return. I am looking forward to you adding this functionality. Thanks. 😊

**Comment by ridicoulous** [comment]
@emin-karadag you can try pre-release version o fnuget package

**Comment by ridicoulous** [comment]
closed by new 1.2.5 release
